android/src/main/java/com/thebluealliance/androidclient/datafeed/status/StatusRefreshService.java

```diff
@@ -52,37 +60,55 @@ protected void onHandleIntent(Intent intent) {
     private void updateTbaStatus() {
         Response<APIStatus> response;
         try {
-             response = mRetrofitAPI.status().toBlocking().first();
-        } catch(Exception ex) {
+            response = mRetrofitAPI.status().toBlocking().first();
+        } catch (Exception ex) {
             Log.w(Constants.LOG_TAG, "Error updating TBA status");
             ex.printStackTrace();
             return;
         }
         if (!response.isSuccessful()) {
-            Log.w(Constants.LOG_TAG, "Unable to update myTBA Status\n"+
-              response.code() + " " +response.message());
+            Log.w(Constants.LOG_TAG, "Unable to update myTBA Status\n" +
+                    response.code() + " " + response.message());
             return;
         }
         APIStatus status = response.body();
 
         /* Write the new data to shared prefs */
         mPrefs.edit()
-          .putString(TBAStatusController.STATUS_PREF_KEY, status.getJsonBlob())
-          .apply();
+                .putString(TBAStatusController.STATUS_PREF_KEY, status.getJsonBlob())
+                .apply();
 
         /* Post the update to the EventBus */
         mEventBus.post(status);
 
+        /* Update Champs pit locations if necessary */
+        if (PitLocationHelper.shouldUpdateFromRemoteUrl(getApplicationContext(), status)) {
+            try {
+                OkHttpClient client = new OkHttpClient();
```

Comment:
Get this from DI. See [`HttpModule`](https://github.com/the-blue-alliance/the-blue-alliance-android/blob/master/android/src/main/java/com/thebluealliance/androidclient/datafeed/HttpModule.java)